config/__autocomplete.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * This class only exists here for IDE (PHPStorm/Netbeans/...) autocompletion.
```

Comment:
Interesting. Could it be achieved with https://www.jetbrains.com/help/phpstorm/ide-advanced-metadata.html? Or is the idea that it may work for any IDE?

Reply:
Did not know about IDE metadata. The goal is the same. Solution in this PR however would work for other than just PHPStorm.